Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh

```diff
@@ -0,0 +1,20 @@
+sudo apt-get -y update
```

Comment:
In order to be considered for inclusion in the public artifact repository, you will to support all variants of linux supported on Azure: Debian Ubuntu, CoreOS, CentOS, SuSE, and Fedora RedHat.  What this means in practical terms is you will need to do the work to determine which package manage is present.  See other public linux artifacts for examples of how to do this.